bigquery/api/SimpleApp/SimpleApp.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
My first impression is that QuickStart and SimpleApp serve the same purpose.  Why are both necessary?

Reply:
We use them in different places in the docs. QuickStart is on language landing pages and library reference pages, and SimpleApp is used for the client library quickstart. Personally, I dislike the one called "QuickStart" and would rather fully replace it with SimpleApp. What do you think @tswast?

Reply:
+1 on removing QuickStart in favor of SimpleApp.

I thought we were going to remove the code sample at https://cloud.google.com/bigquery/docs/reference/libraries (QuickStart) and point to https://cloud.google.com/bigquery/docs/quickstarts/quickstart-client-libraries (SimpleApp) with the new template, anyway?

Reply:
Per in person discussion, I removed the QuickStart sample. If we need something to be explicitly labeled a "QuickStart" in the future, we will rename SimpleApp.